src/computation/logic/quotient-dictionary.ts

```diff
@@ -34,7 +35,7 @@ export class QuotientDictionary extends SortedReverseDict {
         // If more than 1 districts share the lead
         if (winners.length > 1) {
             // Select a winner
-            winner = this.tieBreaker(winners, baseValue);
+            winner = tieBreaker(winners, baseValue);
```

Comment:
A function name should arguably be a verb rather than a noun. "breakTie(winners, baseValue)" for instance. `baseValue` is a little cryptic.

Reply:
Fixed.